invenio_records_lom/marshmallow/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 """Schemas for marshmallow."""
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import, annotations, print_function
 
```

Comment:
for curiosity: why is this necessary here?

Reply:
it allows type-hints to reference not yet defined names (e.g. a class's `.create` method returning an instance of the class itself), which is convenient.
see: https://www.python.org/dev/peps/pep-0563/

plus it's part of the core python language from version 3.11:
https://docs.python.org/3/library/__future__.html

now that I look at that last table..., are `absoulte_import` and `print_function` only necessary for compatibilty with `python2.*`?
we don't support that, do we?

Reply:
yes, we don't support python2.*. those imports can be removed.

yeah but why is the annotations import necessary in __init__.py where no annotations are used?

Reply:
I wrongly thought that `from __future__ import <thing>` would unlock `<thing>` for all files in the sub-file-tree.
It's for this file only though, so I'll remove it.